parser/prism/Translator.cc

```diff
@@ -650,6 +650,23 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
 
             return make_unique<parser::Kwrestarg>(location, sorbetName);
         }
+        case PM_LAMBDA_NODE: { // lambda literals, like `-> { 123 }` or `lambda { 123 }`
```

Comment:
`lambda { 123 }` isn't actually a lambda, as far as the syntax goes. It's a call to `Kernel#lambda` with a blockarg